jetpack.php

```diff
@@ -75,69 +75,88 @@ function jetpack_should_use_minified_assets() {
 	}
 	return true;
 }
-add_filter( 'jetpack_should_use_minified_assets', 'jetpack_should_use_minified_assets', 9 );
```

Comment:
All this red just because wanted to intent it within a conditional block.

Reply:
We don't need to have all this indentation. We can `return` from this include:

```php
if ( version_compare( $GLOBALS['wp_version'], JETPACK__MINIMUM_WP_VERSION, '<' ) ) {
  add_action( 'admin_notices', 'admin_jetpack_unsupported_wp_notice' );
  return;
}

add_filter( 'jetpack_require_lib_dir', 'jetpack_require_lib_dir' );
// ...
```

I don't have a strong opinion either way.

Reply:
Good point.